app/retail/templates/shared/head.html

```diff
@@ -14,9 +14,10 @@
   You should have received a copy of the GNU Affero General Public License
   along with this program. If not, see <http://www.gnu.org/licenses/>.
 {% endcomment %}
-{% load i18n static is_in_list matches %}
+{% load i18n static is_in_list matches compress %}
 {% include 'shared/tag_manager_1.html' %}
 <meta charset="utf-8">
+{% compress css %}
 <link rel="preload" href="{% static "v2/css/base.css" %}" as="style">
```

Comment:
This is a preload tag, so will try to find that file in the server to pre fetch it. Maybe since will be now a big css file, it should preload that file.

Reply:
i guess it depends on if we want it to block the loading of the rest of the site..

Reply:
this will create one file name.css right? I mean <link rel="preload" href="{% static "v2/css/base.css" %}" as="style"> is not the actual call of the file just the instruction to prefetch it. I guess we should have in compress just the files that are loaded in all pages and then add one <link rel="preload" href="{% static "v2/css/compress.css" %}" as="style">

Reply:
> this will create one file name.css right?

yes

> I guess we should have in compress just the files that are loaded in all pages and then add one

i read the [docs on this](https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content) and i dont understand what gain this gives the site.. CSS loading should not block the rendering of theDOM right?  AFAIK it happens ineline

Reply:
so I think is not about blocking is about kb that will load minification + concat. For other side I think the most valuable for us is the minification part since we use http2 and the conections to fetch the diff files will be at the same time.

Reply:
with the preload what happens is the browser knows the priority of the side and will load as soon as possible avoiding any repaint in the browser side

Reply:
here is a good article showing examples of use of preload https://medium.com/reloading/preload-prefetch-and-priorities-in-chrome-776165961bbf